pkg/api/helpers/rate_limit_test.go

```diff
@@ -23,14 +24,48 @@ type HelpersSuite struct{}
 
 var _ = check.Suite(&HelpersSuite{})
 
-func (e *HelpersSuite) TestRateLimit(c *check.C) {
+func (e *HelpersSuite) TestRateLimitBurst(c *check.C) {
 	metricsAPI := mock.NewMockMetrics()
-	limiter := NewApiLimiter(metricsAPI, 10.0, 4)
+	limiter := NewApiLimiter(metricsAPI, 1, 10)
 	c.Assert(limiter, check.Not(check.IsNil))
 
+	// Exhaust bucket (rate limit should not kick in)
 	for i := 0; i < 10; i++ {
 		limiter.Limit(context.TODO(), "test")
 	}
+	c.Assert(metricsAPI.RateLimit("test"), checker.DeepEquals, time.Duration(0))
```

Comment:
Since we are comparing `time.Duration` values (i.e. `int64`) I think `checker.Equals` should be enough.